content/docs/getting-started/adding-a-catalog-item/github/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: GitHub
+publishedDate: '2022-04-04T14:00:00.0Z'
+description: How to configure the GitHub to create a catalog item.
+
+order: 1
+humanName: GitHub
+examples:
+  - name: Component
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: Component
+      metadata:
+        name: artist-web
+        description: The place to be, for great artists
+      spec:
+        type: website
+        lifecycle: production
+        owner: artist-relations-team
+        system: artist-engagement-portal
+        dependsOn:
+          - resource:default/artists-db
+        providesApis:
+          - artist-api
+  - name: Resource
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: Resource
+      metadata:
+        name: artists-db
+        description: Stores artist details
+      spec:
+        type: database
+        owner: artist-relations-team
+        system: artist-engagement-portal
+  - name: API
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: API
+      metadata:
+        name: artist-api
+        description: Retrieve artist details
+      spec:
+        type: openapi
+        lifecycle: production
+        owner: artist-relations-team
+        system: artist-engagement-portal
+        definition: |
+          openapi: "3.0.0"
+          info:
+            version: 1.0.0
+            title: Artist API
+            license:
+              name: MIT
+          servers:
+            - url: http://artist.spotify.net/v1
+          paths:
+            /artists:
+              get:
+                summary: List all artists
+  - name: System
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: System
+      metadata:
+        name: artist-engagement-portal
+        description: Handy tools to keep artists in the loop
+      spec:
+        owner: artist-relations-team
+        domain: artists
+  - name: Domain
+    language: yaml
+    content: |
+      apiVersion: backstage.io/v1alpha1
+      kind: Domain
+      metadata:
+        name: artists
+        description: Everything about artists
+      spec:
+        owner: artist-relations-team
+---
+
```

Comment:
I think we need a short paragraph to introduce the user to what they're about to do.

For example:

> Roadie can populate it's catalog from YAML files that are stored alongside your code and describe the software in each repository. To do this, you need to install our GitHub app, write a YAML file, and import it into Roadie.
> 
> This is the best option if you use GitHub and you want to build your software catalog by adding a YAML metadata file into each repository.

Without this, I can see the different options that are available, but I don't know which I should choose under which circumstances. For example, I have both GitHub and AWS S3 in my company. When would I want to use one vs the other? Should I use both? We should help the reader make decisions like this.

Reply:
Could you re-read this now.